Helper/Ratings.php

```diff
@@ -233,6 +233,20 @@ class_exists('Yotpo\Yotpo\Helper\RichSnippets') &&
         return false;
     }
 
+    /**
+     * Check if the Review module is enabled, review tables are present
+     *
+     * @return bool
+     */
+    public function isReviewModuleEnabled()
```

Comment:
Rename to `canUseMagentoRatingsAndReviews` in order to keep the naming consistent.